packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -369,6 +381,28 @@ export class BlobManager extends TypedEventEmitter<IBlobManagerEvents> {
 		}
 	}
 
+	private async shutdownPendingBlobs(): Promise<void> {
+		for (const [localId, entry] of this.pendingBlobs) {
+			if (entry.status === PendingBlobStatus.OnlinePendingUpload) {
```

Comment:
nit: I think this could be simplified a bit 
```
switch(entry.status){
   case  PendingBlobStatus.OnlinePendingUpload:
      this.sendBlobAttachOp(localId, entry.storageId);
      //fall through to set the handle
  case PendingBlobStatus.OnlinePendingOp:
     entry.handleP.resolve(this.getBlobHandle(localId));
     break; 
  default:
  //nothing to do}
```

Reply:
It can't fall through because I still need to change entry status. I can change the ifs for a switch anyway.

Reply:
but you don't actually change the status, in both cases it gets set to the same value it already is.